webapp/src/client/app/shared/enums/color-format.enum.ts

```diff
@@ -0,0 +1,36 @@
+import { SelectItemWithTooltip } from '@shared/interfaces/select-item-tooltip.interface';
+
+enum ColorFormat {
+  Default = '',
+  Full = 'rgb888le',
+  High = 'rgb655le',
+  Medium = 'rgb332le',
+  Low = 'rgb222le',
+  'Very Low' = 'rgb111le',
+}
+
+enum Tooltips {
+  Default = 'Color format advertised by a server',
+  Full = 'True 24-bit color',
+  High = 'High 16-bit color',
+  Medium = '256 colors',
+  Low = '64 colors',
+  'Very Low' = '8 colors',
```

Comment:
question: Is it really advertised _by_ the server?

Reply:
It's the client's ability to be able to force a specific color format. This one might not be the most commonly used, but it's present in UltraVNC and other clients, so it makes sense to have it as well.